warp-core/src/test/java/com/workday/warp/examples/ExampleTest.java

```diff
@@ -19,11 +22,17 @@ public void vanilla() {
     }
 
 
-    /** An example of using WarpInfoExtension only for the purpose of getting access to a testId. */
-    @WarpInfoProvided
-    public void run(final WarpInfo info) {
-        Logger.trace("we are only being executed");
-        Assert.assertTrue(info.testId().equals("com.workday.warp.examples.ExampleTest.run"));
+    @Test
+    public void testId(final TestInfo info) {
+        final String id = testInfoIsTestId(info).maybeTestId().get();
+        Assert.assertTrue("com.workday.warp.examples.ExampleTest.testId".equals(id));
+    }
+
+
+    @Test
+    @Measure
+    public void measuredonly() {
```

Comment:
should this method be camelCase (i.e measuredOnly)?

Reply:
oops i'll fix that, nice eye